x-pack/plugins/ml/public/application/services/ml_api_service/jobs.ts

```diff
@@ -48,8 +48,8 @@ export const jobsApiProvider = (httpService: HttpService) => ({
     });
   },
 
-  jobs(jobIds: string[]) {
-    const body = JSON.stringify({ jobIds });
+  jobs(jobIds: string[], excludeGenerated?: boolean) {
```

Comment:
As above, did you mean to include these changes? Looks like they are part of the `exclude_generated` work.

Reply:
This is now reverted https://github.com/elastic/kibana/pull/88880/commits/311892828ba8ad9bf92efe643c97c43d7c0439a1